core/controllers/resources_test.py

```diff
@@ -83,38 +83,57 @@ def test_upload_empty_image(self):
 
         self.logout()
 
-    def test_upload_bad_image(self):
```

Comment:
As mentioned above, I've put this test back, given that now we're validating that data passed is actually image data.